spec.html

```diff
@@ -31162,12 +31162,12 @@ <h1>
         </h1>
         <dl class="header">
           <dt>description</dt>
-          <dd>If _sym_ is in the GlobalSymbolRegistry (see <emu-xref href="#sec-symbol.for"></emu-xref>) the String used to register _sym_ will be returned.</dd>
+          <dd>If _sym_ is in the GlobalSymbolRegistry List (see <emu-xref href="#sec-symbol.for"></emu-xref>) the String used to register _sym_ will be returned.</dd>
```

Comment:
```suggestion
          <dd>If _sym_ is in the GlobalSymbolRegistry List, the String used to register _sym_ will be returned.</dd>
```

Now that the term auto-links, we don't need the parenthetical reference.